gateway-js/src/__tests__/executeQueryPlan.test.ts

```diff
@@ -1349,12 +1349,11 @@ describe('executeQueryPlan', () => {
         query,
       });
 
-      expect(response.data).toBeUndefined();
-      expect(response.errors).toMatchInlineSnapshot(`
-        Array [
-          [ValidationError: Cannot query field "ssn" on type "User".],
-        ]
-      `);
+      assert(response.body.kind === 'single');
```

Comment:
Suggestion: unpacking response is done in a few (test) places and it a tad more annoying now. Maybe a simple test helper that extract `data` and `errors` from the response would make things more readable. Fwiw, I just added a [`testUtils.ts`](https://github.com/apollographql/federation/blob/next/gateway-js/src/__tests__/gateway/testUtils.ts) recently for such helpers which could be a place for this.